docs/01-index.md

```diff
@@ -1,29 +1,23 @@
 <img src="{{ path '/images/design-cover-img@2x.jpg' }}" alt="Welcome to design.stjude.cloud!" width="1280px" height="320px" style="max-width: unset"/>
 
 ## Getting Started
-1. If you need to mockup or prototype, you can start by downloading our Sketch DSM - <a href="https://www.sketch.com/s/5f06f430-fa20-4aaf-ad60-6898bbda3de3">Sketch Design System</a>
 
-2. If you are using the theme in your project or prototype simply install the <a href="https://github.com/stjudecloud/theme" target="_blank">theme NPM packages</a>:
+We use Figma for mockups and prototypes on the St. Jude Cloud project. Once you are added to the St. Jude Cloud Figma group, you may visit the [St. Jude Cloud Design System project][figma-dsm]. The instructions to get started with the design system are on the [Getting Started][getting-started-dsm] page in the Figma project. Note that the Figma file may contain experimental components and does not represent the official, production version of our design system.
 
-  * https://github.com/stjudecloud/theme
-  * https://github.com/stjudecloud/theme/packages/140741
+[design.stjude.cloud] and the corresponding source at the [St. Jude Cloud theme repo][theme-repo] represent the production ready version of the design system. Updates to the design system are governed through pull requests that are worked on by multiple teams, and the packages packages listed below are updated in sync to ensure the look and feel remains consistent across the ecosystem.
 
-3. We use <a href="https://getbootstrap.com/" target="_blank">Bootstrap</a> as the base for the St. Jude Cloud theme.
+| Javascript Framework | CSS Framework | NPM Package                                      |
+| -------------------- | ------------- | ------------------------------------------------ |
+| React                | Bootstrap     | See the [official React package][react-package]. |
+| Vue                  | TailwindCSS   | See the [official Vue package][vue-package].     |
```

Comment:
It's my understanding that Viz uses Bootstrap to pull things in from the theme repo and then overlays/converts to Tailwind. Maybe let's add this section when Vue + Tailwind is more formalized in the theme repo?

Reply:
I'll comment this out for now.